docs/source/en/api/pipelines/text_to_video_zero.mdx

```diff
@@ -0,0 +1,154 @@
+    * `t0` and `t1` in the range `{0, ..., num_inference_steps}`. Default: `t0=45`, `t1=48`
+* Video length:
+    * `video_length`, the number of frames video_length to be generated. Default: `video_length=8`
+
+
+### Text-To-Video with Pose Control
+To generate a video from prompt with additional pose control, follow these steps
+
+Read video containing extracted pose images from path
+```python
+import imageio
+
+reader = imageio.get_reader('path/to/your/pose/video',  'ffmpeg')
+frame_count = 8
+pose_images = [Image.fromarray(reader.get_data(i)) for i in range(frame_count)]
+```
+To extract pose from actual video, read [ControlNet documentation](./stable_diffusion/controlnet).
+
+Run `StableDiffusionControlNetPipeline` with our custom attention processor
+
+```python
+import torch
+from diffusers import StableDiffusionControlNetPipeline, ControlNetModel
+from diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_zero import CrossFrameAttnProcessor
+
+model_id = 'runwayml/stable-diffusion-v1-5'
+controlnet = ControlNetModel.from_pretrained("lllyasviel/sd-controlnet-openpose", torch_dtype=torch.float16)
+pipe = StableDiffusionControlNetPipeline.from_pretrained(model_id, controlnet=controlnet, torch_dtype=torch.float16).to('cuda')
+pipe.unet.set_attn_processor(CrossFrameAttnProcessor(batch_size=2))
+
+prompt = 'Darth Vader dancing in a desert'
+result = pipe(prompt=[prompt] * len(pose_images), image=pose_images).images
+imageio.mimsave('video.mp4', result, fps=4)
+```
+
+
+### Text-To-Video with Edge Control
+To generate a video from prompt with additional pose control, follow the steps described above for pose-guided generation using canny edge ControlNet model <br/>
+
+
+### Video Instruct-Pix2Pix
+To perform text-guided video editing, run the following python command:
+
+Read video from path
+```python
+import imageio
+
+reader = imageio.get_reader('path/to/your/video',  'ffmpeg')
+frame_count = 8
+video = [Image.fromarray(reader.get_data(i)) for i in range(frame_count)]
+```
+
+Run `StableDiffusionInstructPix2PixPipeline` with our custom attention processor
+```python
+import torch
+from diffusers import StableDiffusionInstructPix2PixPipeline
+from diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_zero import CrossFrameAttnProcessor
+
+model_id = 'timbrooks/instruct-pix2pix'
+pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(model_id, torch_dtype=torch.float16).to('cuda')
+pipe.unet.set_attn_processor(CrossFrameAttnProcessor(batch_size=3))
+
+prompt = 'edit instruction'
+result = pipe(prompt=[prompt] * len(video), image=video).images
+imageio.mimsave('edited_video.mp4', result, fps=4)
+```
+
+
+### Dreambooth specialization 
+
+Methods **Text-To-Video**, **Text-To-Video with Pose Control** and **Text-To-Video with Edge Control** can run with custom dreambooth models by simply set the `model_id` to corresponding model path or url.
```

Comment:
Could you expand this a bit more with a code snippet? That will be useful for the users.

```suggestion
Methods **Text-To-Video**, **Text-To-Video with Pose Control** and **Text-To-Video with Edge Control** can run with custom [DreamBooth](../training/dreambooth) models by simply set the `model_id` to corresponding model path or URL. You can filter out some available DreamBooth-trained models with [this link](https://huggingface.co/models?search=dreambooth).
```

Reply:
Done!